deps/cjs-module-lexer/src/lexer.c

```diff
@@ -0,0 +1,1246 @@
+#include "cjs-module-lexer.h"
+
+const uint16_t* STANDARD_IMPORT = (uint16_t*)0x1;
+const uint16_t* IMPORT_META = (uint16_t*)0x2;
+const uint16_t __empty_char = '\0';
+const uint16_t* EMPTY_CHAR = &__empty_char;
+// tracked depth of template and brackets
+#define STACK_DEPTH 2048
+// tracked number of star exports
+#define MAX_STAR_EXPORTS 256
+const uint16_t* source;
+
+bool lastSlashWasDivision;
```

Comment:
Using global state in here likely will be problematic. Is this expected to be used in worker threads at all? If so, this is definitely not threadsafe